### ISSUE TITLE
Mapsurfer OSM time out

### ISSUE THREAD

**Issue by image80** [nextgis/quickmapservices#15]
I am working with qgis 2.6.1,

i downloaded the "Quickmap Services" plugin version 0.11. When i try to display these following tiles:
- Mapsurfer OSM Roads Grayscale
- Mapsurfer OSM transparent

an error message appears saying "Download time out: Query network http://129.206.74.245:8008/tms_rg.ashx?x=2&y=0&z=2 (time-out)

it's strange because before, with qgis 2.4, i succeed in displaying thèse tiles.

**Comment by simgislab** [comment]
Is it reproducible? Or you experienced it just once?

**Comment by image80** [comment]
I have just tried again. In vain. Always the same error message. So according to me, unfortunatly, it 's reproductible...;-(

**Comment by simgislab** [comment]
Does it happen with other services?

Is it possible that you have proxy set up in 2.6.1 and not with 2.4?

**Comment by image80** [comment]
Good afternoon,

No, this problem does not happen with other services.

How to know and modify the proxy settings? $

Thank you very much.



2015-02-18 13:28 GMT+01:00 Maxim Dubinin <notifications@github.com>:

**Comment by simgislab** [comment]
check under Settings\Options\Network

**Comment by image80** [comment]
Thanks,

but sorry but I'm not really competent in network side. I just checked the
settings for my two versions (2.4 and 2.6). No proxy parameters is set.
I attached a screen shot representing the qgis network settings.

I succeed in displaying mapsurfer.NET data when i use domestic internet
connexion, at home. But i fail in displaying mapsurfer.NET data via office
professional internet connection.

It's strange because i 'm able to load whatever data from whatever services
providers with professional office internet connection,  except
mapsurfer.NET data
And  i would like to use this one (for the transparent and the road
grayscale).

2015-02-18 13:45 GMT+01:00 Maxim Dubinin <notifications@github.com>:
